android/start.md

```diff
@@ -1,116 +1,277 @@
+Build an Android app using the AWS Amplify CLI and the AWS SDK for Android. The Amplify CLI lets you quickly add backend features to your application so that you can focus on your application code. This page guides you through setting up an initial backend and integration into your app. 
 
-## Step 0: Set Up Your Development Environment
+## Prerequisites
 
-We strongly recommend that you use the Amplify CLI for building the serverless backend for your app. If you have already installed the CLI, skip ahead to [Step 2](./add-aws-mobile-sdk-basic-setup).
+[Install Android Studio](https://developer.android.com/studio/index.html#downloads) version 2.33 or higher. Install Android SDK for API level 23 (Android SDK 6.0).
+
+Install the Amplify CLI. If you have already installed the CLI, skip ahead to [Step 2](./add-aws-mobile-sdk-basic-setup).
 
 *  [Sign up for an AWS Account](https://portal.aws.amazon.com/billing/signup?redirect_url=https%3A%2F%2Faws.amazon.com%2Fregistration-confirmation#/start).
 
-*  Install [Node.js](https://nodejs.org/) and npm (if they are not already installed).
+*  Install [Node.js](https://nodejs.org/) and [npm](https://www.npmjs.com/get-npm) if they are not already installed.
 
-> Verify that you are running at least Node.js version 8.11.x or greater and npm version 5.x or greater by running `node -v` and `npm -v` in a terminal/console window. Older versions aren't supported and might generate errors.
+Verify that you are running at least Node.js version 8.11+ or greater and npm version 5.x or greater by running `node -v` and `npm -v` in a terminal/console window.
+{: .callout .callout--action}
 
 To install and configure the Amplify CLI globally, run the following commands in a terminal window.
 
-  ```bash
-   $ npm install -g @aws-amplify/cli
-   $ amplify configure
-   ```
-
-Minimum requirements for your development environment are as follows.
+Install and configure the Amplify CLI.
 
-* Choose the Android Java app project you want to integrate with an AWS backend.
+```bash
+$ npm install -g @aws-amplify/cli
+$ amplify configure
+```
 
-* [Install Android Studio](https://developer.android.com/studio/index.html#downloads) version 2.33 or higher.
+Note: These commands will install the CLI globally. If you're using Windows, the CLI currently supports <a href="https://docs.microsoft.com/en-us/windows/wsl/install-win10" target="_blank">Windows Subsystem for Linux</a>.
+{: .callout .callout--action}
 
-* Install Android SDK for API level 23 (Android SDK 6.0).
 
 ## Step 1: Create a new app
 
-* Follow [these steps](https://developer.android.com/training/basics/firstapp/creating-project) to create a new Android Studio application
+Follow [these steps](https://developer.android.com/training/basics/firstapp/creating-project) to create an Android Studio application using Java. Modify your `project/build.gradle` with the following dependency in the build script:
```

Comment:
build dependency. This can cause confusion.

Reply:
Updated.